libs/datamodel/core/src/ast/reformat/reformatter.rs

```diff
@@ -422,7 +422,10 @@ impl<'a> Reformatter<'a> {
         } else {
             vec!["id", "unique", "index", "map"]
         };
-        return if let Some(sort_index) = correct_order.iter().position(|p| directive_name.contains(p)) {
+        return if let Some(sort_index) = correct_order
```

Comment:
`return` is not needed here, you can remove the keyword and the semicolon in the end of the expression.